src/box/sql/where.c

```diff
@@ -4625,7 +4625,7 @@ sqlWhereBegin(Parse * pParse,	/* The parser context */
 			} else if (iAuxArg
 				   && (wctrlFlags & WHERE_OR_SUBCLAUSE) != 0) {
 				iIndexCur = iAuxArg;
-				op = OP_IteratorReopen;
+				assert(op == OP_IteratorOpen);
```

Comment:
Thank you for the review!

This place was reachable, however the only thing the `op` variable was used for was checking that it was not equal to `0`. Since OP_IteratorReopen opcode is only mentioned here, this leads to a situation where this opcode becomes unused.